python/src/MethodOfMomentsFactory_doc.i.in

```diff
@@ -14,17 +14,41 @@ Notes
 -----
 This method fits a scalar distribution to data of dimension 1, using the method of moments.
 
-Let :math:`(\vect{x}_1, \dots, \vect{x}_n)` denote the sample, :math:`F_{\vect{\theta}}`
+Let :math:`\vect{x} = (\vect{x}_1, \dots, \vect{x}_n)` denote the sample, :math:`F_{\vect{\theta}}`
 the  cumulative distribution function we want to fit to the sample,
-and :math:`\vect{\theta} \in  \Theta \subset\Rset^p`  its parameter vector.
+and :math:`\vect{\theta} \in  \Theta \subset\Rset^K`  its parameter vector.
 
-Let :math:`K` denote the number of parameters of the distribution and we assume that 
+Then :math:`K` denotes the number of parameters of the distribution and we assume that
 the :math:`K` first moments of the distribution exist.
 
-Let :math:`(\mu_1, \dots, \mu_K)` denote the :math:`K` first central moments of the sample and
-and :math:`(m_1, \dots, m_K)` those of the parametric model.
+Let :math:`(m_1, \dots, m_K)` those coming from the parametric model which
```

Comment:
The current text is a mixture of $m_1$ and $\mu_1$. I suggest to stick with $\mu_1$. 
```suggestion
Let :math:`(\mu_1, \dots, \mu_K)` those coming from the parametric model which
```

Reply:
ok